srcpkgs/libomp/template

```diff
@@ -0,0 +1,34 @@
+pkgname=libomp
+version=7.0.0
+revision=1
+build_style=cmake
+configure_args="-DLIBOMP_ENABLE_SHARED=ON -G Ninja"
+make_cmd="ninja"
+hostmakedepends="ninja perl pkg-config"
+makedepends="elfutils-devel libffi-devel"
+short_desc="LLVM OpenMP Runtime Library"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="Custom"
+homepage="https://openmp.llvm.org"
+distfiles="https://releases.llvm.org/$version/openmp-$version.src.tar.xz"
+checksum=30662b632f5556c59ee9215c1309f61de50b3ea8e89dcc28ba9a9494bba238ff
+wrksrc="openmp-$version.src"
+
+do_install() {
+	cd ${cmake_builddir:=build}
+	DESTDIR=${DESTDIR} ninja install
+}
+
+post_install() {
+	vlicense LICENSE.txt
+	# libgomp.so is also provided by libgomp; remove it to fix the conflict
+	rm -f ${DESTDIR}/usr/lib/libgomp.so
```

Comment:
Should problably just be included in the -devel package.

libgomp is provided by libgomp-devel, make them conflict ?